test/XRP/xrp-protocol-buffer-conversion.test.ts

```diff
@@ -159,14 +161,39 @@ describe('Protocol Buffer Conversion', function (): void {
   })
 
   it('Convert IssuedCurrency with bad value', function (): void {
-    // GIVEN an issued currency protocol buffer with a non numeric value
-    // WHEN the protocol buffer is converted to a native TypeScript type.
-    const issuedCurrency = XrpIssuedCurrency.from(
-      testInvalidIssuedCurrencyProto,
+    // GIVEN an issued currency protocol buffer with an invalid value field
+    // WHEN the protocol buffer is converted to a native TypeScript type THEN an error is thrown.
+    assert.throws(
+      () => {
+        XrpIssuedCurrency.from(testInvalidIssuedCurrencyProtoBadValue)
+      },
+      XrpError,
+      'Cannot construct BigInt from IssuedCurrency protobuf `value` field.',
```

Comment:
this feels pretty brittle to me, as it means that changes to the log text will result in a test failure. i think there's a couple solves here

1) honestly, i think it's probably ok to not check the error message
2) from what i can tell, the main reason we need to do this is because XrpError doesn't encode enough information. you could do something where you implement a new error type that has enough information...honestly this feels like overkill right now

i would go with 1

Reply:
Alright, I'll take them out.